packages/jokul/src/components/card/index.ts

```diff
@@ -1,7 +1,7 @@
-export { Card } from "./Card.js";
-export { CardImage } from "./CardImage.js";
+export { Card, type CardPadding, type CardVariant } from "./Card.js";
+export { CardImage, type CardImageProps } from "./CardImage.js";
 
 export { NavCard, InfoBlock, type NavCardProps } from "./NavCard.js";
 export { TaskCard, type TaskCardProps } from "./TaskCard.js";
 export { InfoCard, type InfoCardProps } from "./InfoCard.js";
-export type { SpacingStep, PaddingOptions } from "./types.js";
```

Comment:
Er `PaddingOptions` eksportert noe sted nå? Den er forskjellig fra `CardPadding` som brukes i det nye kortet, så den bør være tilgjengelig separat (i hvert fall i de gamle kortene).

Hvis vi vil sørge for at de ikke blir brukt i det nye kortet kan vi vurdere å namespace dem i de gamle komponentene:

```ts
// i NavCard.tsx
import type { PaddingOptions } from "./types.js";

export namespace NavCard {
    export type Padding = PaddingOptions;
}
```

Da er de tilgjengelige som `NavCard.Padding` der du har importert den komponenten.

Reply:
Ah, så ikke at den var i bruk direkte. Fant bare andre typer som extender den i `InfoCard`, `NavCard` og `TaskCard`.

Hvis det ikke er nok at _de_ er eksportert kan jeg fint legge dette tilbake.

Reply:
Ah, ok! Det kan godt hende det bare er de som er i bruk, altså! Men er kanskje best å legge tilbake for sikkerhets skyld, i tilfelle noen har tatt utgangspunkt i basevarianten.

Reply:
Men det er jo ingenting i base-varianten som bruker dem?

Reply:
Nei, mente `PaddingOptions` som "basevariant" av de du snakket om som extendet den i `NavCard` et al

Reply:
Forstår fremdeles ikke helt men da legger jeg den tilbake.